scripts/check_compliance.py

```diff
@@ -252,7 +263,7 @@ def parse_kconfig(self):
         os.environ['GENERATED_DTS_BOARD_CONF'] = "dummy"
 
         # For multi repo support
-        get_modules(os.path.join(tempfile.gettempdir(), "Kconfig.modules"))
+        self.get_modules(os.path.join(tempfile.gettempdir(), "Kconfig.modules"))
```

Comment:
Could get rid of the `"Kconfig.modules"` argument and inline it into the function maybe.